apps/platform-app/src/app/[locale]/(profile)/_workflows/prompts/evaluateProfileNode.ts

```diff
@@ -4,28 +4,99 @@ import {
   HumanMessagePromptTemplate,
```

Comment:
**suggestion**: Consider breaking down the `systemMessage` into smaller parts for better readability.